modules/kuberay-cluster/kuberay-autopilot-values.yaml

```diff
@@ -156,108 +157,15 @@ worker:
   # If you want to disable the default workergroup
   # uncomment the line below
   disabled: true
-  groupName: workergroup
```

Comment:
why'd we wipe this?

Reply:
Because Ray Autoscaler and worker groups are disabled on the GKE Standard setup.